campari/run_one_object.py

```diff
@@ -0,0 +1,443 @@
+# Standard Library
+import pathlib
+import warnings
+
+# Common Library
+import galsim
+import numpy as np
+import scipy.sparse as sp
+from astropy.utils.exceptions import AstropyWarning
+from erfa import ErfaWarning
+from numpy.linalg import LinAlgError
+from roman_imsim.utils import roman_utils
+
+# SN-PIT
+from snpit_utils.config import Config
+from snpit_utils.logger import SNLogger
+
+from campari.data_construction import construct_images, prep_data_for_fit
+from campari.model_building import construct_static_scene, construct_transient_scene, generate_guess, make_grid
+
+# Campari
+from campari.simulation import simulate_images
+from campari.utils import banner, get_weights
+
+# This supresses a warning because the Open Universe Simulations dates are not
+# FITS compliant.
+warnings.simplefilter("ignore", category=AstropyWarning)
+# Because the Open Universe Sims have dates from the future, we supress a
+# warning about using future dates.
+warnings.filterwarnings("ignore", category=ErfaWarning)
+
+r"""
+Cole Meldorf 2024
+Adapted from code by Pedro Bernardinelli
+
+                    ___
+                   / _ \___  __ _  ___ ____
+                  / , _/ _ \/  ' \/ _ `/ _ \
+                 /_/|_|\___/_/_/_/\_,_/_//_/
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣔⣴⣦⣔⣠⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣼⣿⣭⣿⣟⣿⣿⣿⣅⢀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣾⣷⣾⣿⣿⣿⣿⣿⣿⣿⡶⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣄⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇⠄⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣠⠤⢤⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⡠⢒⣿⣿⣿⣠⠋⠀⠀⠀⠀⠀⠀⣀⣀⠤⠶⠿⠿⠛⠿⠿⠿⢻⢿⣿⣿⣿⠿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⡞⢀⣿⣿⣿⡟⠃⠀⠀⠀⣀⡰⠶⠛⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⠀⠃⠘⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠘⢧⣤⣈⣡⣤⠤⠴⠒⠊⠉⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+
+
+                 _____  __     ___  __________
+                / __/ |/ /    / _ \/  _/_  __/
+               _\ \/    /    / ___// /  / /
+              /___/_/|_/    /_/  /___/ /_/
+
+
+"""
+# Global variables
+huge_value = 1e32
```

Comment:
Not conceptual point of PR, but I have questions and concerns about the need and purpose of this global.